modules/dists/SparseBlockDist.chpl

```diff
@@ -279,8 +283,10 @@ class LocSparseBlockDom {
   param rank: int;
   type idxType;
   param stridable: bool;
+  type sparseLayoutType = DefaultDist;
```

Comment:
It might be better not to specify the default here in LocSparseBlockDom, since we probably need to always set it explicitly and probably any code in Block.chpl that relies on this default is an error.